config/site/src/_includes/filtering_predicate_definitions/conjunctions.md

```diff
@@ -6,11 +6,11 @@
   filters that can't be provided on a single filter input because of collisions between key names.
   For example, if you want to provide multiple `anySatisfy: ...` filters, you could do `allOf: [{anySatisfy: ...}, {anySatisfy: ...}]`.
 
-  Will be ignored when `null` or an empty list is passed.
+  Will be treated as `true` when `null`, an empty object or list is passed.
 
 [`anyOf`]({% link query-api/filtering/conjunctions.md %})
 : Matches records where any of the provided sub-filters evaluate to true.
   This works just like an `OR` operator in SQL.
 
-  Will be ignored when `null` is passed. When an empty list is passed, will
-  cause this part of the filter to match no documents.
+  Will be treated as `true` when `null` or an empty object is passed.
```

Comment:
Similar comment here about empty object: there's a gotcha related to that and we don't want to encourage empty objects to be passed so maybe not worth mentioning?

Reply:
Makes sense, I have removed the mention of empty object.